Packs/Threat_Crowd/Integrations/ThreatCrowd_v2/ThreatCrowd_v2.yml

```diff
@@ -0,0 +1,325 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: ThreatCrowd_v2
+  version: -1
+configuration:
+- defaultvalue: https://www.threatcrowd.org/searchApi/v2/
+  display: Server
+  name: server_url
+  required: true
+  type: 0
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: C - Fairly reliable
+  display: Source Reliability
+  name: integrationReliability
+  options:
+  - A+ - 3rd party enrichment
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+- additionalinfo: How many entries to fetch. For full data use -1.
+  defaultvalue: '50'
+  display: Max Number of Entries
```

Comment:
```suggestion
  display: Max Number of resolution  and hashes to retrieve.
```

Reply:
This also includes subdmains when running !domain. Maybe number of results per indicator?